agent1: {
      simulator: {
            publish_frequency: 10.0,
            dt: 0.005,
            gps_freq_update: 1000.0,
            lowLevelDyn: False,
            delay_df: 0.0,
            delay_a: 0.0,
            n_bound_pr: 2.0,
            x_std_pr: 0.0034,
            y_std_pr: 0.0034,
            psi_std_pr: 0.0034,
            vx_std_pr: 0.01,
            vy_std_pr: 0.0,
            psiDot_std_pr: 0.01,
            n_bound: 2.0,
            x_std: 0.04,
            y_std: 0.04,
            psi_std: 0.04,
            vx_std: 0.01,
            vy_std: 0.00,
            psiDot_std: 0.01,
            du_0: 0.1,
            du_1: 0.05,
            u_bound: 1.0,
            dist_mode: False,
            mu_sf: 0.1,
            Cd: 0.020,
            A_car: 0.03,
            init_x: 0.0,
            init_y: 1.0,
            init_yaw: 0.0,
            init_vx: 0.5,
            init_vy: 0.0,
            init_ax: 0.0,
            init_ay: 0.0,
      },
      
      control : {
            mode: simulations,
            saveData: true,
            N: 20,
            Hz: 30,
            vel_ref: 1.5,
            vel_th: 0.0,
            lane: 0
      }
      
}

agent2: {
      simulator: {
            publish_frequency: 10.0,
            dt: 0.005,
            gps_freq_update: 1000.0,
            lowLevelDyn: False,
            delay_df: 0.0,
            delay_a: 0.0,
            n_bound_pr: 2.0,
            x_std_pr: 0.0034,
            y_std_pr: 0.0034,
            psi_std_pr: 0.0034,
            vx_std_pr: 0.01,
            vy_std_pr: 0.0,
            psiDot_std_pr: 0.01,
            n_bound: 2.0,
            x_std: 0.04,
            y_std: 0.04,
            psi_std: 0.04,
            vx_std: 0.01,
            vy_std: 0.00,
            psiDot_std: 0.01,
            du_0: 0.1,
            du_1: 0.05,
            u_bound: 1.0,
            dist_mode: False,
            mu_sf: 0.1,
            Cd: 0.020,
            A_car: 0.03,
            init_x: 0.0,
            init_y: 2.0,
            init_yaw: 0.0,
            init_vx: 0.25,
            init_vy: 0.0,
            init_ax: 0.0,
            init_ay: 0.0,
      },

      control: {
            mode: simulations,
            saveData: true,
            N: 20,
            Hz: 30,
            vel_ref: 1.5,
            vel_th: 0.3,
            lane: 1
      }
}